layouts/partials/footer.html

```diff
@@ -1,5 +1,6 @@
 <footer class="footer">
   <section class="container">
+    <p>{{ .Site.Params.footercontent }}</p>
```

Comment:
I think it's better to use:
```
{{ with .Site.Params.footercontent }}
  <p>{{ . }}</p>
{{ end }}
```
to prevent empty `<p>` if `footercontent` is missing or empty.